cfme/tests/cloud_infra_common/test_power_control_manual.py

```diff
@@ -254,7 +260,6 @@ def test_power_operations_from_global_region():
             5.
             6. Power operations applicable for this vm should be working as expected
     """
-    pass
```

Comment:
I think I would like to require this `pass` statement on manual test cases

Reply:
since I have two votes against I obey :)